README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
+Defaults to the version set in `postgresql::server`.  Use this if you want to manage `pg_hba.conf` without managing the entire postgreSQL instance.
+
+####`manage_pg_hba_conf`
```

Comment:
Seems like my original idea was wrong but I've updated the PR with my understanding.  Let me know what you think.